File: Java/混/01.始/02.枚/枚举.java
/* 枚举
*/
class 周{
    enum 星期{日, 一,二,三,四,五,六}
    星期 几;
}
public class 枚举{
    public static void main(String[] args) {
        周 周几 = new 周();
        周几.几 = 周.星期.三;

        System.out.println("星期几？ "+周几.几);
    }
}

File: cpp/cpp2e/ch03/枚举.cpp
#include <iostream>
#include <string>
using namespace std;

enum class 周 : char
{
    一 = '1',
    二,
    三,
    四,
    五,
    六,
    日
};

int main()
{
    cout << (char)(周::一) << endl;
}